setup.py

```diff
@@ -469,10 +477,11 @@ def check_cudnn_version_and_warn(global_option: str, required_cudnn_version: int
 
     # CUDA group norm supports from SM70
     arch_flags = []
-    for arch in [70, 75, 80, 86, 90]:
+    # FIXME: this needs to be done more cleanly
+    for arch in [70, 75, 80, 86, 90, 100, 120]:
         arch_flag = f"-gencode=arch=compute_{arch},code=sm_{arch}"
         arch_flags.append(arch_flag)
-    arch_flag = f"-gencode=arch=compute_90,code=compute_90"
+    arch_flag = f"-gencode=arch=compute_120,code=compute_120"
```

Comment:
I believe this `arch_flag` will end up with duplicated `sm_120`:
```
['-gencode=arch=compute_70,code=sm_70',
 '-gencode=arch=compute_75,code=sm_75',
 '-gencode=arch=compute_80,code=sm_80',
 '-gencode=arch=compute_86,code=sm_86',
 '-gencode=arch=compute_90,code=sm_90',
 '-gencode=arch=compute_100,code=sm_100',
 '-gencode=arch=compute_120,code=sm_120',
 '-gencode=arch=compute_120,code=compute_120']
```
I do not think that is intended. I believe it is safe to remove that `arch_flag = f"-gencode=arch=compute_120,code=compute_120"` line.

Reply:
`code=compute_120` is intended to be used on future GPUs with higher SM versions. For example, when using a future GPU with compute capability 13.0 and a proper CUDA driver, the compute_120 PTX code will be compiled to code_130 with nvrtc.

Reply:
I am not sure if we really need this, we can follow-up on adding new sm version whenever there is a new hardware is out. Why are not not doing this `+PTX` on the other APEX components?